src/RadioButtons/index.tsx

```diff
@@ -20,7 +14,7 @@ interface RadioButtonsProps {
    *   | string;
    * ```
    * */
-  options: Record<string, OptionType>;
+  options?: object;
```

Comment:
I would like to revisit this, but in the meantime this looser type prevents type errors throughout the staff app